axelar/assetlist.json

```diff
@@ -1249,6 +1249,47 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/yieldeth.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/yieldeth.svg"
       }
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ox-wei",
+          "exponent": 0,
+          "aliases": [
+            "0x78a0A62Fba6Fb21A83FE8a3433d44C73a4017A6f"
+          ]
+        },
+        {
+          "denom": "ox",
+          "exponent": 18
+        }
+      ],
+      "base": "ox-wei",
+      "name": "Open Exchange Token",
+      "display": "ox",
+      "symbol": "OX",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0x78a0A62Fba6Fb21A83FE8a3433d44C73a4017A6f"
```

Comment:
Please register this asset in Ethereum's assetlist file.

Reply:
added